packages/components/psammead-media-player/src/Guidance/index.test.jsx

```diff
@@ -8,6 +8,16 @@ describe('Media Player: Guidance', () => {
     <Guidance
       guidanceMessage="Guidance: Contains strong language with adult humor"
       service="news"
+      type="video"
+    />,
+  );
+
+  shouldMatchSnapshot(
+    'should render no-js styles when noJsClassName prop is used',
+    <Guidance
+      guidanceMessage="Guidance: Contains strong language with adult humor"
+      service="news"
+      type="video"
```

Comment:
You're not passing in the `noJsClassName` prop here. 

It can be easy for updates to be made and the snapshot changes missed. I suggest adding an assertion test - so when passing in prop `noJsClassName`, the styles rendered include class `.${noJsClassName}`.

Reply:
done.